### ISSUE TITLE
Can we have a Twitter Ad blocker?

### ISSUE THREAD

**Issue by AppleAuthority** [AdguardTeam/AdGuardForSafari#163]
My friend has this with his Ad Blocker (AdBlocker Ultimate) and I'm really jealous

**Comment by ameshkov** [comment]
Well, AdGuard for Safari should block Twitter ads just okay. Isn't it in your case?

Also, AdBlocker Ultimate is a simple rip-off of our old Safari extension.

**Comment by AppleAuthority** [comment]
No, it doesn't. I'm worried if I create a filter it will block everything in my timeline

**Comment by ameshkov** [comment]
What filters do you have enabled? Could it be that you hit the 50k limit?

**Comment by AppleAuthority** [comment]
It says I have three filters and 32744 rules

**Comment by ameshkov** [comment]
@slavaleleka could you please take a look?

**Comment by slavaleleka** [comment]
@AppleAuthority if AdGuard Base filter enabled there is no ad on my end — 
`twitter.com##.promoted-trend` blocks it. So, first of all, try to add this rule to user filter

**Comment by AppleAuthority** [comment]
I did and nothing happened. I also tried reinstalling it, and it really looks like it's trying
![Screen Recording 2019-05-23 at 9 51 20 am](https://user-images.githubusercontent.com/44805409/58239458-fdbe5900-7d40-11e9-8dfe-03239c7dbc1d.gif)

**Comment by zzebrum** [comment]
@AppleAuthority can you check in incognito mode, please? Can't repro either.

**Comment by AppleAuthority** [comment]
Did, and I also disabled my other extensions. Nothing

**Comment by TheHasagi** [comment]
@AppleAuthority 

Please, update filters and try again.

**Comment by AppleAuthority** [comment]
@TheHasagi It's...Trying...So...Hard!
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/44805409/59423525-398d9100-8dca-11e9-872f-a27090bd70cd.gif)

**Comment by zzebrum** [comment]
@AppleAuthority have you tried to delete cookies for twitter?

**Comment by AppleAuthority** [comment]
No, why would that help?
